packages/plugin-goat/package.json

```diff
@@ -1,23 +1,25 @@
 {
-  "name": "@elizaos/plugin-goat",
-  "version": "0.1.7-alpha.1",
-  "main": "dist/index.js",
-  "type": "module",
-  "types": "dist/index.d.ts",
-  "dependencies": {
-    "@elizaos/core": "workspace:*",
-    "@goat-sdk/core": "0.3.8",
-    "@goat-sdk/plugin-erc20": "0.1.7",
-    "@goat-sdk/wallet-viem": "0.1.3",
-    "@goat-sdk/plugin-coingecko": "0.1.4",
-    "tsup": "8.3.5",
-    "viem": "2.21.53"
-  },
-  "scripts": {
-    "build": "tsup --format esm --dts",
-    "dev": "tsup --format esm --dts --watch"
-  },
-  "peerDependencies": {
-    "whatwg-url": "7.1.0"
-  }
+    "name": "@elizaos/plugin-goat",
+    "version": "0.1.7-alpha.1",
+    "main": "dist/index.js",
+    "type": "module",
+    "types": "dist/index.d.ts",
+    "dependencies": {
+        "@elizaos/core": "workspace:*",
+        "@goat-sdk/adapter-vercel-ai": "~0.2.0",
```

Comment:
Probably need to make these versions static to avoid potential supply chain attacks, right? @odilitime

Reply:
Fixed! :)